### PR TITLE
Call kubetest directly instead of e2e-runner.sh

### PR DESCRIPTION
**Pull request by fejta**
ref https://github.com/kubernetes/test-infra/issues/2829
Also add (an increasingly noisy) warning to stop using e2e-runner.sh before August.

/assign @rmmh @ixdy

### CODE REVIEW

File: jenkins/e2e-image/e2e-runner.sh [hunk 1]
```diff
@@ -22,34 +22,18 @@ set -o xtrace
...
+echo 'Call kubetest directly' >&2
 
-e2e_go_args=()
-# TODO(fejta): set KUBETEST_MANUAL_DUMP in kubernetes_e2e.py after new image
-if [[ -z "${KUBETEST_MANUAL_DUMP:-}" ]]; then
-  e2e_go_args=( \
-    -v \
-    --dump="${WORKSPACE}/_artifacts" \
-  )
+if [[ "$(date +%s)" -gt "$(date --date='August 1 2017')"]]; then
```
Comment (ixdy) [inline jenkins/e2e-image/e2e-runner.sh]: is the second `date` missing a `+%s`?

also on the next line.

File: jenkins/e2e-image/e2e-runner.sh [hunk 2]
```diff
@@ -22,34 +22,18 @@ set -o xtrace
 
 export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
-# TODO(fejta): consider moving this elsewhere
-echo "--------------------------------------------------------------------------------"
-echo "Test Environment:"
-printenv | sort
-echo "--------------------------------------------------------------------------------"
+echo 'WARNING: e2e-runner.sh is deprecated! This will error on August 2nd' >&2
+echo 'Call kubetest directly' >&2
```
Comment (ixdy) [inline jenkins/e2e-image/e2e-runner.sh]: link to an issue, maybe?

File: scenarios/kubernetes_e2e.py [hunk 3]
```diff
@@ -201,13 +178,12 @@ def use_latest_image(self, image_family, image_project):
         print >>sys.stderr, 'Set KUBE_GCE_NODE_PROJECT=%s' % image_project
 
     def start(self, args):
-        """Runs e2e-runner.sh after setting env and installing prereqs."""
+        """Runs runner after setting env and installing prereqs."""
```
Comment (ixdy) [inline scenarios/kubernetes_e2e.py]: `s/runner/kubetest/`?

also we no longer install prereqs.

Comment (fejta) [inline scenarios/kubernetes_e2e.py]: Done

File: jenkins/e2e-image/e2e-runner.sh (hunk 1 above)
Comment (fejta) [inline jenkins/e2e-image/e2e-runner.sh]: Done

File: jenkins/e2e-image/e2e-runner.sh (hunk 2 above)
Comment (fejta) [inline jenkins/e2e-image/e2e-runner.sh]: Done

**Pull request merged**